.github/workflows/cmake.yml

```diff
@@ -6,12 +6,32 @@ on:
   pull_request:
     # The branches below must be a subset of the branches above
     branches: [ main ]
+    #Changes to the following files do not trigger deployment
+    paths-ignore:
+     - README.md
+     - 'os_stub/openssllib/openssl/**'
+     - 'os_stub/mbedtlslib/mbedtls/**'
+     - 'unit_test/cmockalib/cmocka/**'
 
 env:
   BUILD_TYPE: Release
 
 jobs:
-  build:
+  format_check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: Check code format
```

Comment:
Please add a comment that this is checking for tabs. This will work for now, but later on we'll probably want something more sophisticated, and we may have files where tabs are required (Python, YAML, etc).

Reply:
Thanks. I have added a comment. And I have made the Tab check better, which the check only run on *.c/*.h file. And the check will skip on openssl/mbedtls/cmocka.